letsencrypt/views/base_config_settings.xml

```diff
@@ -0,0 +1,19 @@
+<odoo>
+
+    <record id="base_config_settings" model="ir.ui.view">
+        <field name="name">Letsencrypt base config settings</field>
+        <field name="model">base.config.settings</field>
+        <field name="inherit_id" ref="base_setup.view_general_configuration" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[last()]" position="after">
+                <group name="letsencrypt" string="LetsEncrypt">
+                    <field name="dns_provider" />
+                    <field name="dns_function" />
```

Comment:
i think it's a good idea to group fields by dns provider, and make their visibility depend on the value of the selection field. you could also do nifty stuff like checking the domains the user wants certificates for, if there's a wildcard among it, make the dns provider field required, and every dns provider will itself inject fields with the suitable `attrs` dictionary. also a big plus on extendability, think of a hidden boolean field like `letsencrypt_needs_dns_provider` which checks for the wildcards, then all dns providers can make their fields required (if necessary) when the flag is true and the selection field has their key as value